src/listener/osx_keyboard_listener.rs

```diff
@@ -0,0 +1,40 @@
+use livesplit_hotkey::{Hook, Hotkey, KeyCode, Modifiers};
+use std::{error::Error, thread};
+
+use common::KeyboardListenerProvider;
+
+use super::common;
+pub struct OSXKeyboardListener {
+    hook: Hook,
+    hotkey: Hotkey,
+    callback: fn(),
+}
+
+impl KeyboardListenerProvider for OSXKeyboardListener {
+    fn new(callback: fn()) -> Result<OSXKeyboardListener, Box<dyn Error>> {
+        return Ok(OSXKeyboardListener {
+            hook: Hook::new().unwrap(),
+            hotkey: Hotkey {
+                key_code: KeyCode::KeyC,
+                modifiers: Modifiers::META,
+            },
```

Comment:
why are these the settings for "new" ? are they just placeholders? if so we should note that or use default values (by implementing the Default trait)